README.md

```diff
@@ -1,18 +1,50 @@
+
+Note that the Health (FHIR / Argonaut) APIs for access to Veteran health records are not yet in production; dev access in sandbox using mock patient data is available, and we expect these APIs to be in production by fall of 2019.
 
 ## Available Services
+\* APIs marked with an asterisk are currently for internal VA use only.
+
+### Address Validation*
+- The [Address Validation](https://developer.va.gov/explore/verification/docs/address_validation) service provides methods to both standardize and validate addresses.
+
+### Appeals Status*
+- Use the [Appeals Status](https://developer.va.gov/explore/benefits/docs/appeals) API to request the status of a Veteran's benefits claim appeal. 
+
+### Benefits Claims
+- Use the [Claims](https://developer.va.gov/explore/benefits/docs/claims) (Auto-Establishment) API to submit a Veteran's benefits claim via EVSS (Electronic Veterans Self-Service), and to request the status of a Veteran's benefits claim.
+
+### Benefits Intake (fka Document Upload Service)
+- The [Benefits Intake](https://developer.va.gov/explore/benefits/docs/benefits) API allows authorized third-party systems used by Veteran Service Organizations and agencies to upload claim documents (scanned to PDF) directly to the Veterans Benefits Administration's (VBA) claims intake process.
+
+### Facilities
+- Use the [Facilities](https://developer.va.gov/explore/facilities/docs/facilities) API to find contact info, location, hours of operation, available services and other relevant information about a specific VA facility. 
+
+### Health (Dev only)
+- Use the [Health APIs](https://developer.va.gov/explore/health/docs/argonaut) to allow Veterans to view their medical records, schedule an appointment, find a specialty facility, and share their information with caregivers and providers.
+
+### Veteran Verification
+
+- The [Veteran Verification](https://developer.va.gov/explore/verification) API empowers Veterans to access their service history, Veteran status, discharge information and disability rating and put that information to work for them.
+
+#### Disability Rating
+- The [Disability Rating](https://developer.va.gov/explore/verification/docs/disability_rating) endpoint allows third-parties to access the disability rating of a Veteran after receiving authorization to do so using an Open ID Connect flow.
+
+#### Service History
+- The [Service History](https://developer.va.gov/explore/verification/docs/service_history) endpoint allows third-parties to access the service history of a Veteran after receiving authorization to do so using an Open ID Connect flow.
+
+#### Veteran Confirmation
+
+- The [Veteran Confirmation](https://developer.va.gov/explore/verification/docs/veteran_confirmation) endpoint allows third-parties to request confirmation from the VA of an individual's Veteran status after receiving authorization to do so using an Open ID Connect flow.
```

Comment:
Why is this README duplicating content from the developer portal? IMO linking to the developer portal seems more sustainable.

Reply:
@dvogel I was extrapolating from the original service offered, which was only VBA document uploads (aka Benefits Intake API).  Given that several devs have come in this way even recently, a short summary with a link to the dev portal docs didn't seem out of line.